models/payout.go

```diff
@@ -15,31 +14,45 @@ type Payout struct {
 func AllocatePayouts() error {
 	currentTime := time.Now()
 
-
-	payouts, ledgerEntries, err := calculatePayouts(currentTime)
+	ledgerEntries, err := calculatePayouts(currentTime)
 	if err != nil {
-		Log.Errorf("Error calculating payouts: %v", err)
```

Comment:
Any reason why we're killing the log here?

Reply:
this doesn't kill the log, it just logs the err with with a error code

Reply:
Na I mean't removing the Log.Errorf line entirely in this PR. Killing was in reference to the removal of the line, not to the functionality.

Reply:
ah lol. duh, tee hee.